phpstan.neon

```diff
@@ -24,7 +24,7 @@ parameters:
           path: pinc/3rdparty/mediawiki/DairikiDiff.php
         - message: '#Comparison operation .* results in an error#'
           path: pinc/3rdparty/mediawiki/DiffEngine.php
-        - message: '#^Parameter \#2 \$num of function array_fill expects int, float given\.$#'
+        - message: '#^Parameter \#2 \$.* of function array_fill expects int, float given\.$#'
```

Comment:
That's a small mistake I made in the previous PR that makes PHPStan fails: PHP 8 does actually see the error but with a different parameter name.